pulser-pasqal/pulser_pasqal/backends.py

```diff
@@ -152,7 +150,9 @@ class EmuFreeBackend(PasqalEmulator):
     Args:
         sequence: The sequence to send to the backend.
         connection: An open PasqalCloud connection.
-        config: An EmulatorConfig to configure the backend.
+        config: An EmulatorConfig to configure the backend. If not provided,
+            the default configuration is used.
     """
 
     emulator = pasqal_cloud.EmulatorType.EMU_FREE
+    default_config = DEFAULT_CONFIG_EMU_FREE
```

Comment:
No configurable fields for EMU_FREE?

Reply:
Only the default for now (`"backend_options"`), since the only field `EmuFreeConfig` currently has is `with_noise`.